virtual_rainforest/models/animals/model_schema.json

```diff
@@ -22,12 +22,27 @@
                             "diet": {
                                 "type": "string",
                                 "default": ""
+                            },
+                            "metabolic_type": {
+                                "type": "string",
+                                "default": ""
+                            },
+                            "birth_mass": {
+                                "type": "number",
+                                "default": 0.0
+                            },
+                            "adult_mass": {
+                                "type": "number",
+                                "default": 0.0
                             }
```

Comment:
What does it mean if these values are zero?

Reply:
It means I've put in a default value without thinking it through!

Reply:
Yeah if you provide default values missing values get filled when the schema gets validated, if you don't then the schema validation fails if missing values are encountered for that field. I'm guessing you probably want the validation to fail if someone supplies a taxon without a body mass? If so I would just delete the default field here

Reply:
Agreed ^^